saleor/graphql/order/tests/test_fulfillment.py

```diff
@@ -1811,13 +1812,28 @@ def test_fulfillment_approve_delete_products_before_approval_allow_stock_exceede
     response = staff_api_client.post_graphql(
         query, variables, permissions=[permission_manage_orders]
     )
-    content = get_graphql_content(response, ignore_errors=True)
-    errors = content["errors"]
-    assert len(errors) == 1
-    assert (
-        errors[0]["message"]
-        == "Insufficient stock for Test product (SKU_AA), Test product 2 (SKU_B)"
-    )
+
+    content = get_graphql_content(response)
+    errors = content["data"]["orderFulfillmentApprove"]["errors"]
+
+    assert len(errors) == 2
+    error_field_and_code = {
+        "field": "stocks",
+        "code": "INSUFFICIENT_STOCK",
+    }
+    expected_errors = [
+        {
+            **error_field_and_code,
+            "message": "Insufficient product stock: Test product (SKU_AA)",
+        },
+        {
+            **error_field_and_code,
+            "message": "Insufficient product stock: Test product 2 (SKU_B)",
```

Comment:
I know that it was like that before, but we should get the product name and SKU from instances.